src/bors/command/parser.rs

```diff
@@ -70,6 +70,27 @@ impl<'a> Tokenizer<'a> {
 
 type ParseResult<'a> = Option<Result<BorsCommand, CommandParseError<'a>>>;
 
+
+//  parse a line of text into command and key-value pairs
+fn parse_key_value(line: &str) -> Option<(&str, Vec<(&str, &str)>)> {
```

Comment:
@Kobzol We might also have to modify tokenizer and other functions to handle this change right ?

Reply:
Yes. In fact I would rather get rid of the tokenizer and just write this using some parsing library (`chomsky`, `nom` etc.) to provide useful error messages. Or, we should implement error reporting by hand.

With the existing implementation, if there's a single "syntax error", the whole command will be just `None`, so we will not be able to provide useful feedback to users on what went wrong.

Reply:
IMO we could go with hand written error reporting, would give us much more control

Reply:
If you're up to it, feel free to implement it! :)